config/schedule.rb

```diff
@@ -197,12 +203,14 @@
   options = {}
   options[:at] = task[:at] if task[:at].present?
   every task[:frequency], options do
-    if ENV['ECS'] == 'true' && task[:interruptable]
-      rake_short task[:task]
-    else
-      # For the time being, move all cron tasks to the "short-term" capacity provider
-      rake_short task[:task]
-      # rake task[:task]
-    end
+    # if ENV['ECS'] == 'true' && task[:interruptable]
```

Comment:
Why are these  commented out?

Reply:
See comment above re rubocop

Reply:
This is used during deployment, not sure why rubocop would think it's not in use (it definitely is).

Reply:
Sorry I wasn't clear, this is a different style violation. In this case, both branches of the conditional are the same (Style/IdenticalConditionalBranches). So the above is equivalent to just
`rake_short task[:task]`